src/routes/embedded/auth-request/connect/connect-custom.view.tsx

```diff
@@ -59,20 +55,10 @@ export function WalletSettingsCustomEmbeddedView() {
     [handlePermissionChange],
   );
 
-  const handleCancel = async () => {
```

Comment:
Now, accepting or cancelling an auth request will not automatically close the popup. Instead, resolving the last one will redirect users back to `/wallet`, and it will be the SDK who devices if the popup is closed or not (if it was opened automatically, it will; if it was opened before the auth request came in by the user, then it will stay open).